Jasmine/Grid/ViewControllers/GridGameViewController.swift

```diff
@@ -57,9 +61,11 @@ class GridGameViewController: UIViewController {
     /// Listens to a drag gesture and handles the operation of dragging a tile, and dropping it
     /// to another location.
     ///
-    /// This also starts the game if have not done so.
+    /// Note that if the game status is not in progress, results in no-op.
```

Comment:
Updated documentation. Prefer to state what the method does at the start, rather than stating the pre-condition first.